contracts/strategy/StrategyBaseV2.sol

```diff
@@ -127,8 +134,9 @@ abstract contract StrategyBaseV2 is IStrategyV2, ControllableV3 {
 
     _emergencyExitFromPool();
 
-    address _asset = asset;
+    address _asset = asset; // gas saving
     uint balance = IERC20(_asset).balanceOf(address(this));
+    _decreaseBaseAmount(_asset, balance); // TODO probably we should reset base-amount here?
```

Comment:
yes, we should just reset
emergency exit for critical purposes and assume no assets stay in the strategy